plugins/box-whiskers.js

```diff
@@ -27,27 +33,17 @@
                 var xs = screenModel.model.scaleX;
                 var ys = screenModel.model.scaleY;
                 var model = {
-                    x: function (d) {
```

Comment:
I recommend you to use ES5 function style in plugins.
-----------------
(cite from slack)
We use old ES5 standard in a plugins ecosystem since we want plugins to be loosely coupled with library core and work without a transpilers or in a browsers without ES6 support. Also we have a plans to extract them to separate repositiory in future
-----------------